src/ray/raylet/scheduling_queue.cc

```diff
@@ -223,6 +223,32 @@ void SchedulingQueue::QueueMethodsWaitingForActorCreation(
   QueueTasks(methods_waiting_for_actor_creation_, tasks);
 }
 
+bool SchedulingQueue::TaskInQueue(const TaskID &task_id) const {
```

Comment:
can we change the method name to be `HasTask` (i.e., make it consistent with the related TaskQueue method name)